trp_project_scrum/tests/test_trp_project_scrum.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp.tests.common import TransactionCase
+
+
+class TestTrpProjectScrum(TransactionCase):
+        def setUp(self):
+            self.collaborator = self.env['res_users'].create({
+                'name' : 'collaborator',
+                'groups_id': [(
+                    6, 0 , self.env.ref('trp_project_scrum_collaborators')
+                )
+            })
+
+            self.collaborator_follower = self.env['res_users'].create({
+                'name' : 'collaborator',
+                'groups_id': [(
+                    6, 0 ,
+                    self.env.ref('trp_project_scrum_collaborators_followers')
+                )
+            })
+
+            self.project1 = self.env['project.project'].create({
+                'active' : True,
+                'analytic_account_id' : 1,
+                'state': open
+            })
+
+            self.project2 = self.env['project.project'].create({
+                'active' : True,
+                'analytic_account_id' : 1,
+                'state': open
+            })
+
+            
+            self.project3 = self.env['project.project'].create({
+                'active' : True,
+                'analytic_account_id' : 1,
+                'state': open
+            })
+        
+            # Add tasks to projects
+
+            # add followers and members
+
+
+        def test_permissions(self):
+            # Assign a task to user
```

Comment:
those are missing badly, especially tests proving that external users don't see records they're *not* supposed to see